website/docs/r/lb_virtual_server.html.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_lb_virtual_server"
+sidebar_current: "docs-vcd-resource-lb-virtual-server"
+description: |-
+  Provides an NSX edge gateway load balancer virtual server resource.
+---
+
+# vcd\_lb\_virtual\_server
+
+Provides a vCloud Director Edge Gateway Load Balancer Virtual Server resource. Adds an edge gateway
+internal or uplink interface as a virtual server. A virtual server has a public IP address and services all incoming client requests. 
+
+~> **Note:** To make load balancing work one must ensure that load balancing is enabled on edge gateway (edge gateway must be advanced).
+This depends on NSX version to work properly. Please refer to [VMware Product Interoperability Matrices](https://www.vmware.com/resources/compatibility/sim/interop_matrix.php#interop&29=&93=) 
+to check supported vCloud director and NSX for vSphere configurations.
+
+~> **Note:** The vCloud Director API for NSX supports a subset of the operations and objects defined in the NSX vSphere 
+API Guide. The API supports NSX 6.2, 6.3, and 6.4.
+
+Supported in provider *v2.4+*
+
+## Example Usage (HTTP Virtual Server)
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_virtual_server" "http" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+  
+  name       = "http-virtual-server"
+  ip_address = "1.1.1.1" # Edge Gateway uplink interface IP
+  protocol   = "http"    # Must be the same as specified in application profile
+  port       = 80
+  
+  app_profile_id = "${vcd_lb_app_profile.http.id}"
+  server_pool_id = "${vcd_lb_server_pool.web-servers.id}"
+  app_rule_ids   = ["${vcd_lb_app_rule.redirect.id}", "${vcd_lb_app_rule.language.id}"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
+when connected as sysadmin working across different organisations
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `edge_gateway` - (Required) The name of the edge gateway on which the virtual server is to be
+created
+* `name` - (Required) Virtual Server name
+* `description` - (Optional) Virtual Server description
```

Comment:
Sometimes you use "Virtual Server" and sometimes "virtual server". Please chose one in user-facing messages.

Reply:
Lowercase it will be then :)